draft-packages/menu/KaizenDraft/Menu/components/MenuContent.module.scss

```diff
@@ -1,23 +1,23 @@
-@import "~@kaizen/design-tokens/sass/color";
-@import "~@kaizen/design-tokens/sass/color-vars";
-@import "~@kaizen/design-tokens/sass/border";
-@import "~@kaizen/design-tokens/sass/shadow";
-@import "~@kaizen/design-tokens/sass/typography";
+@import "~@kaizen/design-tokens/sass/border-vars";
+@import "~@kaizen/design-tokens/sass/shadow-vars";
+@import "~@kaizen/design-tokens/sass/spacing-vars";
+@import "~@kaizen/design-tokens/sass/typography-vars";
 @import "~@kaizen/deprecated-component-library-helpers/styles/layout";
 @import "~@kaizen/component-library/styles/layers";
 
-$side-padding: 1/2 * $ca-grid;
-$gutter: 4px;
+$side-padding: calc(1 / 2 * #{$kz-var-spacing-md});
+$gutter: calc(#{$kz-var-spacing-md} * 1 / 6);
+$padding: calc(#{$kz-var-spacing-md} * 1 / 4);
```

Comment:
You can use `$kz-spacing-xs` here - this equates to 1.5rem (ca-grid/spacing-md) * 0.25 = 0.375rem (`$kz-spacing-xs`)

`side-padding` can also be changed to `$kz-spacing-sm`

Reply:
ah nice thank you!